modules/ocf_backups/files/create-encrypted-backup

```diff
@@ -23,30 +39,40 @@ fi
 cleanup() {
     if [ -e "$SNAPSHOT_PATH" ]; then
         echo "Removing logical volume: $SNAPSHOT_PATH"
-        lvremove -f "$SNAPSHOT_PATH"
+        /sbin/lvremove -f "$SNAPSHOT_PATH"
```

Comment:
Yeah, seems like a good idea, especially since the default path used is `/usr/bin:/bin`, but files like `/etc/crontab` set the path to `/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin`. It looks like there is [some discussion](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=617898) of changing it but that it probably won't happen soon.

I don't see there being much trouble if this moved though, this script would just fail for one week (at the `lvcreate` stage) and then be fixed soon after that with the new path. It's still less portable though, so I agree that setting the `PATH` globally would be a good idea. Plus it avoids the need for absolute paths for everything not in `/usr/bin`or `/bin`, which is good.